src/Media/Audio/OpenALSoundProvider.cpp

```diff
@@ -793,7 +793,7 @@ PAudioDataSource PlatformDataSourceInitialize(PAudioDataSource baseDataSource) {
 
 class AudioSample16 : public IAudioSample {
  public:
-    AudioSample16():al_source(-1) {}
+    AudioSample16():al_source(-1), _position(0.0, 0.0, 0.0), _maxDistance(0.0), _volume(0.0) {}
```

Comment:
Just move these into inline initialization below? And `Vec3f` is initialized to zero in ctor anyway.